apps/wallet-mobile/src/yoroi-wallets/cardano/constants/common.ts

```diff
@@ -54,7 +54,7 @@ export const LINEAR_FEE = {
 } as const
 export const MINIMUM_UTXO_VAL = '1000000'
 
-export const HISTORY_REFRESH_TIME = 25000
+export const HISTORY_REFRESH_TIME = 35000
```

Comment:
avg block is ~20s, doing below 30s has a high chance to sync with no updates, ideal 30/worst 40/avg 35s. (suggesting to add as a config)